.openpublishing.publish.config.json

```diff
@@ -27,7 +27,7 @@
   ],
   "sync_notification_subscribers": [],
   "branches_to_filter": [],
-  "git_repository_branch_open_to_public_contributors": "main",
+  "git_repository_branch_open_to_public_contributors": "live",
```

Comment:
@cincuranet I think this alone is supposed to work. What were you getting when you tried it?